### ISSUE TITLE
cannot fix georss

### ISSUE THREAD

**Issue by ses0043** [Automattic/jetpack#13467]
#### Steps to reproduce the issue

1. In W3C Feed Validation https://validator.w3.org/feed/ 
2. I type my rss feed link comfortcruising.com/feed or comfortcruising.com/feed/rss
3. Click check

#### What I expected
My RSS should work and have no errors

#### What happened instead
The validator replies line 9, column 97: XML parsing error: <unknown>:9:97: duplicate attribute 
xmlns:georss="http://www.georss.org/georss" xmlns:geo="http://www.w3.org/2003/01/geo/wgs84_pos#" xmlns:georss="http://www.georss.org/georss" xmlns:geo="http://www.w3.org/2003/01/geo/wgs84_pos#" xmlns:ymaps="http://api.maps.yahoo.com/Maps/V2/AnnotatedMaps.xsd" >

and my RSS does not work

**Comment by jeherve** [comment]
You seem to be using 2 different Geo Location solutions; one is provided by Jetpack, the other by the WP Geo plugin.

If you would rather only use the WP Geo plugin, you can deactivate Jetpack's Geo Location features by adding the following code snippet to your site:

```php
function jetpackcom_remove_geo( $tools ) {
    unset( $tools[ array_search( 'geo-location.php', $tools, true ) ] );
    return $tools;
}
add_filter( 'jetpack_tools_to_include', 'jetpackcom_remove_geo' );
```

**Comment by ses0043** [comment]
Thank you so much for your quick response. I added your recommended
code to both the child theme function file and still the same errors.
Any other recommendations would be appreciated.
Kind regards

**Comment by ses0043** [comment]
Today I received an email from WordPress letting me know that my site was down. I removed the Geo RSS code that you recommended and I still have the same issue. Thanks for your help in advance.

**Comment by jeherve** [comment]
> I added your recommended code to both the child theme function file 

Could you try to add it to one place only, and try to add it to a functionality plugin like this one instead:
https://wordpress.org/plugins/code-snippets/

Let me know how it goes.

**Comment by ses0043** [comment]
Thank you again for your quick response. I removed the code from both child and theme function file. I installed the Code Snippets plugin and cloned the "order snippet by name" changed the name, replaced the code with your recommended code, selected the option "run snippet everywhere", left the priority a 10 and saved changes. Using W3 Total Cache cleared all cache. 
Feed validator still indicates the geo RSS has a duplicate entry.
